provision-cluster/delete.js

```diff
@@ -1,14 +1,30 @@
+'use strict';
 const core = require('@actions/core');
 const github = require('@actions/github');
+const kubeception = require('./lib/kubeception.js');
 
 try {
   // inputs are defined in action metadata file
   const distribution = core.getInput('distribution');
   const version = core.getInput('version');
-  console.log(`Deleting ${distribution} ${version}!`);
   // Get the JSON webhook payload for the event that triggered the workflow
   //const payload = JSON.stringify(github.context.payload, undefined, 2)
   //console.log(`The event payload: ${payload}`);
+
+  const clusterName = process.env['clusterName']
+
+	switch(distribution) {
+	   case "Kubeception": {
```

Comment:
Should the distribution be case sensitive? I'm betting someone at some point will write `kubeception`